cartoframes/carto_vl/defaults.py

```diff
@@ -0,0 +1,25 @@
+_HTML_TEMPLATE = '<iframe srcdoc="{srcdoc}" width="{width}" height="{height}"></iframe>'
+
+_CARTO_VL_PATH = 'https://libs.cartocdn.com/carto-vl/v1.2.4/carto-vl.min.js'
+
+_AIRSHIP_SCRIPT = '/packages/components/dist/airship.js'
+_AIRSHIP_BRIDGE_SCRIPT = '/packages/bridge/dist/asbridge.js'
+_AIRSHIP_STYLE = '/packages/styles/dist/airship.css'
+_AIRSHIP_ICONS_STYLE = '/packages/icons/dist/icons.css'
+_AIRSHIP_COMPONENTS_PATH = 'https://libs.cartocdn.com/airship-components/v2.0/airship.js'
+_AIRSHIP_BRIDGE_PATH = 'https://libs.cartocdn.com/airship-bridge/v2.0/asbridge.js'
+_AIRSHIP_STYLES_PATH = 'https://libs.cartocdn.com/airship-style/v2.0/airship.css'
+_AIRSHIP_ICONS_PATH = 'https://libs.cartocdn.com/airship-icons/v2.0/icons.css'
+
+_CREDENTIALS = {
```

Comment:
Why do we need this default? I feel that if we need this it should be defined in the specs for testing-only. I don't see how one of our accounts is useful as a default to the user.

Reply:
You're right, let's leave it empty 👍